cellarium/cas/resources/low_quota_feedback_template.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    <hr/>
+    <p>You only have {remaining_quota} cells left in your quota.  You can increase your quota to 200k by providing feedback.</p>
```

Comment:
How about saying something happier?

```
Thank you for using CAS! Please note that you have {remaining_quota} cells remaining in your quota during the beta testing phase.

We’d love to hear your feedback on your experience so far and how we can improve the service. As a token of appreciation, we will extend your free quota to 200,000 cells as soon as we receive your input.
```

Can you also ensure that these messages render nicely, with the appropriate amount of blank lines below and above? Thanks!

Reply:
Will do!